src/lib/logical_query_plan/lqp_translator.hpp

```diff
@@ -12,7 +12,7 @@ class AbstractOperator;
 class TransactionContext;
 
 /**
- * Translates an LQP (Logical Query Plan), represented by its root node, into an Operator tree for the execution
+ * Translates a LQP (Logical Query Plan), represented by its root node, into an Operator tree for the execution
```

Comment:
I think `an` was correct here (only before `LQP`, not before `Logical Query Plan`), since it is [phonetically based.](https://english.stackexchange.com/a/378281)